frontend/lib/ui/artefact_page/artefact_page.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:yaru_widgets/yaru_widgets.dart';
+
+import '../../providers/family_artefacts.dart';
+import '../../routing.dart';
+import '../dialog_header.dart';
+import '../spacing.dart';
+import 'artefact_page_body.dart';
+import 'artefact_page_header.dart';
+import 'artefact_page_info_section.dart';
+
+class ArtefactPage extends ConsumerWidget {
+  const ArtefactPage({super.key, required this.artefactId});
+
+  final int artefactId;
+
+  Column get _invalidArtefactErrorMessage {
+    return const Column(
+      children: [
+        DialogHeader(),
```

Comment:
Given that we don't have a Dialog anymore, do we still need to render the error message in a Dialog box, or we can just maybe render it on the page itself?

Reply:
UX wise, I imagine there are two kinds of errors. First is when a user was supposed to navigate to a certain page but something went wrong. Second is when a user is just making some kind of action on the same page without navigating. I think it makes sense in the first case to navigate to an error page. But in the second case perhaps it's more user friendly to just have a dialog. wdyt?